tests/unit_tests/lcls_tools/common/data/test_archiver.py

```diff
@@ -613,17 +678,37 @@ def side_effect(pv_list, time_stamp):
 
         mocked_get_data.side_effect = side_effect
 
+        interval = get_data_with_time_interval(
+            self.pv_lst,
+            self.time - timedelta(days=10),
+            self.time,
+            timedelta(days=1),
+        )
         self.assertEqual(
-            get_data_with_time_interval(
-                self.pv_lst,
-                self.time - timedelta(days=10),
-                self.time,
-                timedelta(days=1),
-            ),
+            interval,
             self.expected_time_delta_result,
         )
 
+    # TODO Fix actual Assertion error (ArchiveDataHandler objects are different)
     def test_get_values_over_time_range_with_timedelta(self):
+        # Trying to debug
+        """
```

Comment:
Can you delete these if they're not needed anymore?

Reply:
Yep!